### PR TITLE
[13.0] l10n_pe: change round method for Peru

### PR DESCRIPTION
**Pull request by umiphos**
Description of the issue/feature this PR addresses:

Changing the rounding method for new companies, it should be round per line and not globally, this is done because the logic of the accounting in Peru for the calculation of the taxes demands it.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by umiphos** [comment]
cc @nhomar @jco-odoo

File: addons/l10n_pe/models/res_company.py [hunk 1]
```diff
@@ -8,6 +8,6 @@ class ResCompany(models.Model):
 
     @api.onchange('country_id')
     def onchange_country(self):
-        """In Peru, the rounding method it's calculated as global"""
+        """In Peru, the rounding method it's calculated as round_per_line"""
         for rec in self.filtered(lambda x: x.country_id == self.env.ref('base.pe')):
-            rec.tax_calculation_rounding_method = 'round_globally'
+            rec.tax_calculation_rounding_method = 'round_per_line'
```
Comment (jco-odoo) [inline addons/l10n_pe/models/res_company.py:13]: Better to remove this code.  It is round_per_line by default.

File: addons/l10n_pe/models/account_chart_template.py [hunk 2]
```diff
@@ -9,6 +9,6 @@ def _load(self, sale_tax_rate, purchase_tax_rate, company):
         """Load the rounding method properly to the company once the CoA is installed.
         """
         company.filtered(lambda c: c.country_id.id == self.env.ref('base.pe').id).write({
-            'tax_calculation_rounding_method': 'round_globally',
+            'tax_calculation_rounding_method': 'round_per_line',
```
Comment (jco-odoo) [inline addons/l10n_pe/models/account_chart_template.py:12]: same here, better to remove it

**Comment by umiphos** [comment]
@jco-odoo deleted the rounding method from l10n_pe, anymore comments here?

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**